docs/index.md

```diff
@@ -33,12 +34,12 @@ class SubModel(BaseModel):
 
 
 class Settings(BaseSettings):
-    auth_key: str = Field(validation_alias='my_auth_key')
-    api_key: str = Field(validation_alias='my_api_key')
+    auth_key: str = Field('', validation_alias='my_auth_key')
+    api_key: str = Field('', validation_alias='my_api_key')
```

Comment:
I think the idea here was that these fields were required - then set via env vars. Probably best if you can set the appropriate env vars in the test so this works without modifiation.

Reply:
Done